.travis.yml

```diff
@@ -51,12 +51,18 @@ before_script:
   # only the arm64 fallback repo is unsigned and needs --allow-unauthenticated
   - sudo apt-get -y --allow-unauthenticated install librados-dev librbd-dev
 
+# Two stages for testing, each stage runs its jobs in parallel, but stages are
+# run after each other, unless the last stage fails.
+# Only run the deploy stage for the master branch on push (not pull_request)
+# events.
+stages:
+  - build testing
+  - e2e testing
+  - name: deploy
+    if: branch = master AND type = push
```

Comment:
can we remove branch Comparision just check its a PR or push?

Reply:
Removed now. I added it as it was part of the original `deploy:` section.